examples/chip-tool-darwin/commands/pairing/PairingCommandBridge.mm

```diff
@@ -99,3 +102,54 @@
                          setupPINCode:mSetupPINCode
                                 error:error];
 }
+
+void PairingCommandBridge::Unpair()
+{
+    dispatch_queue_t callbackQueue = dispatch_queue_create("com.chip-tool.command", DISPATCH_QUEUE_SERIAL);
+    [CurrentCommissioner()
+        getConnectedDevice:mNodeId
+                     queue:callbackQueue
+         completionHandler:^(CHIPDevice * _Nullable device, NSError * _Nullable error) {
+             CHIP_ERROR err = CHIP_NO_ERROR;
+             if (error) {
```

Comment:
Please merge the error handling here on top of https://github.com/project-chip/connectedhomeip/pull/18427 ?

Reply:
Done